src/screens/HomePage/HomePage.tsx

```diff
@@ -36,7 +44,7 @@ function getPageDescription() {
 export default function HomePage() {
   const shareBlockRef = useRef(null);
   const location = useLocation();
-  const [showCounties, setShowCounties] = useState(false);
+  const [locationScope, setLocationScope] = useState('States');
```

Comment:
rather than hard-coding the strings here and down below in the component, we can use a string [enum](https://www.typescriptlang.org/docs/handbook/enums.html), so-

```
  enum MapView {
    STATES = 'States', 
    COUNTIES = 'Counties'
  }

  const [locationScope, setLocationScope] = useState<MapView>(MapView.STATES);
...
...
...
  <Button value={MapView.STATES}>States</Button>
  <Button value={MapView.COUNTIES}>Counties</Button>
```